src/htm/algorithms/SpatialPooler.cpp

```diff
@@ -500,6 +500,11 @@ void SpatialPooler::compute(const SDR &input, const bool learn, SDR &active) {
 
 void SpatialPooler::boostOverlaps_(const vector<SynapseIdx> &overlaps, //TODO use Eigen sparse vector here
                                    vector<Real> &boosted) const {
+  if(boostStrength_ < htm::Epsilon) {
+    const auto begin = static_cast<const SynapseIdx*>(overlaps.data());
+    boosted.assign(begin, begin + overlaps.size());
+    return; //boost ~ 0.0, we can skip these computations. 
```

Comment:
this probably changed the results for the test

Reply:
Can you be a little more certain of this?  Why exactly did the unit tests change?

Reply:
Nevermind, I see how your unit test works.

Reply:
nitpick: `boosted.assign(overlaps.begin(), overlaps.end())`